test/util/bitcoin-util-test.py

```diff
@@ -77,7 +77,7 @@ def bctest(testDir, testObj, buildenv):
     are not as expected. Error is caught by bctester() and reported.
     """
     # Get the exec names and arguments
-    execprog = os.path.join(buildenv["BUILDDIR"], "src", testObj["exec"] + buildenv["EXEEXT"])
+    execprog = os.path.join(os.environ.get('BITCOINPATH', '') or os.path.join(buildenv["BUILDDIR"], "src"), testObj["exec"] + buildenv["EXEEXT"])
```

Comment:
Hmm, this env var is not documented and makes the code a bit harder to read. Would it be too ugly to call `mkdir src && mv ./bitcoin*.exe ./src` in the appveyor after compile?